core/common/src/main/java/alluxio/util/webui/NodeInfo.java

```diff
@@ -39,7 +39,12 @@
    * @param workerInfo the worker info
    */
   public NodeInfo(WorkerInfo workerInfo) {
-    mHost = workerInfo.getAddress().getHost();
+    if (!workerInfo.getAddress().getContainerHost().equals("")) {
+      mHost = workerInfo.getAddress().getHost() + "("
+              + workerInfo.getAddress().getContainerHost() + ")";
```

Comment:
string concat is expensive. Use `String.format`?

Reply:
good idea,thsnks